docs/create-manage-organizations.md

```diff
@@ -42,15 +42,15 @@ Here are the different permissions available for these roles:<br />
 
 ## Add organization members
 
-Administrators can add any number of Technical users to the organization and even add other Admins if necessary. <br /><br />
-To add a user to your organization:
+Administrators can add any number of Technical users to the organization, and even add other Admins if necessary. <br /><br />
+To add users to your organization:
 
 1. Log in to [Percona Platform](https://portal.percona.com).
 2. Click **Organization** tab on the left side menu. 
 3. On the **Members** tab, click the **Invite Member** button. 
-4. Enter your team member's corporate email address and select a role.
-5. Click **Save** to send the invitation to join your organization. 
-
+4. Enter one or more email addresses for the team member you want to invite, and select a role.
```

Comment:
I think here it's better to say "Enter email address for the.." because we can't specify more than one email in 1 line.